### PR TITLE
#163388090 Setup environment for testing

### PR DESCRIPTION
**Pull request by Arusey**
**What does this PR do?**
It sets up the environment for writing tests for the application

**Description of the Tasks completed?**
-Installed enzyme package: a testing utility for react applications
-Created the setupTest.js file which integrates enzyme with react and jest

**How should this be manually tested?**

-Clone this repository `https://github.com/andela/ah-the-phoenix-frontend.git`

-cd into `ah-the-phoenix-frontend`

-checkout to the branch `ch-test-enviroment-163388090`

-run `npm install`

-when complete run `npm run test`

-choose *option a* to run all tests

image

image

**Relevant Pivotal Tracker [Stories**]
[#163388090](https://www.pivotaltracker.com/n/projects/2240061)

### CODE REVIEW

**Comment by ezrqnkemboi** [comment]
@Arusey we are expected to only add FTL review needed badge after devs have reviewed your work. 
Also, only one badge is required at a time. That is after devs finish their reviews, you remove devs label and put FTL.

File: src/App.test.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
-  ReactDOM.unmountComponentAtNode(div);
+ describe('App', ()=>{
+  it('Should render without crashing', function(){
+    const app = shallow(<App/>)
+    expect(app.length).toBe(1)
+
+   });
 });
+
+describe('Addition', () => {
```
Comment (ezrqnkemboi) [inline src/App.test.js]: Let's remove this test. The first one can serve as an example. We may merge this and later forget to delete it, which brings more issues.

**Comment by dev-jey** [comment]
The testing environment looks good to me

File: src/App.test.js (hunk 1 above)
Comment (Arusey) [reply src/App.test.js]: Noted working on it right now

**Comment by ezrqnkemboi** [comment]
In the future, we can also be updating our screenshots and PR descriptions after making some changes, if it is required. 
@Arusey, you can change the label for this PR.

File: .circleci/config.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ jobs:
           command: |
             curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
             chmod +x ./cc-test-reporter
+            collectCoverageFrom='!src/serviceworker.js
```
Comment (ezrqnkemboi) [inline .circleci/config.yml]: Remove this for now

**Pull request merged**